.github/pull_request_template.md

```diff
@@ -14,16 +14,17 @@ Please check all options that are relevant
 - [ ] Documentation update only  
 
 # How Has This Been Tested?  
-
 **Tested Configuration**:  
-- [ ] CentOS, Specify version(s):  
-- [ ] PostgreQL, Specify version(s):  
-- [ ] docs tested with hugo <0.60  
+- [ ] Ansible, Specify version(s):  
+- [ ] Specify Distro & version(s):  
+- [ ] Specify Repo Distro & version(s):  
```

Comment:
What does repo distro/version mean in context with pgMonitor?

Reply:
I assume the same as w/ paar. Are you using the CentOS rpms from us or the RHEL rpms from us. Which probably doesn't make a lot of sense, since they could be using upstream source, yes?

Reply:
Ahh ok. Thought asking the Distro would answer this question, but forgot they could possibly be using centos repo on rhel or vice versa. Yes, they could possibly be using upstream for Prometheus/Grafana. But I guess we'd want to know that as well. 

Maybe clarify that you mean package repository?

Reply:
how's that @keithf4 ?

Reply:
Looks good

Reply:
should i do the same for paar then? i think 'yes'

Reply:
"Packages compiled from source" suggests to me something different than I think you're intending, and is at least ambiguous.  Rephrase?

Reply:
what do you think it meand and what do you think I intend @pgguru ?

Reply:
@hunleyd see my suggestions for reworking this area to get the idea of how it might be disambiguated.